modules/javafx.graphics/src/test/java/test/com/sun/javafx/pgstub/StubToolkit.java

```diff
@@ -929,4 +959,14 @@ public List<MenuBase> getMenus() {
     public GlassRobot createRobot() {
         throw new UnsupportedOperationException("Not supported yet.");
     }
+
+    /**
+     * Ensures that the current (test) runs with the StubToolkit.
+     * Throws a class cast exception otherwise.
+     *
+     * @return the StubToolkit instance
+     */
+    public static StubToolkit ensure() {
+        return (StubToolkit)Toolkit.getToolkit();
+    }
```

Comment:
This is unused (and may not be needed). I think you can revert the changes to this file.